### DISCUSSION TITLE
phalcon 3,4 webhook problem

### DISCUSSION THREAD

**Discussion by faroukdon** [phalcon/phalcon#408] [Q&A]
i have a webhook which i receive data from it , i run a function responsable for extracting data from that webhook and send a response , it takes to much within 20 min without sending the reposne while the 3 rd parti thinks that there is no response and it triggers again the webhook
in a normal way my function takes 11 sec to be executed when i run the function and replay the webhook from ngrok it works perfectly , also when i eliminate all my code and send only response when i receive the webhook it works perfectly

phalcon 3,4 
linux
ngrok
Uplisting (webhook 3rd partie
![wewe](https://github.com/phalcon/phalcon/assets/49798442/3756100f-2e7e-4293-8f82-3f6aa0fc2e5e)
)

**Comment by kgrammer** [comment]
@faroukdon, Since you are just asking for coding help, your question needs to be posted on a general coding site like StackOverflow.com.  

Github is used for posting specific Phalcon bug reports and asking for new Phalcon features.  
This issue is likely to get closed as an incomplete bug report here.

**Comment by faroukdon** [comment]
thanks
